meta-builder/Makefile

```diff
@@ -27,11 +27,10 @@ run: generate fmt vet manifests
 
 # Install CRDs into a cluster
 install: manifests
-	kubectl apply -f config/crd/bases
+	kustomize build config/crd | kubectl apply -f -
```

Comment:
kustomize is part of kubectl.  Should use kubectl commands.

Reply:
Currently, our repo since it's scaffolded through kubebuilder requires `kustomize`, at least for installing the crd (https://book.kubebuilder.io/quick-start.html#installation).  For example, line 35 in the Makefile is using `kustomize` also.  This is simply a port of an upstream patch.  If we want to remove `kustomize` completely, we can do that in another PR.